README.md

```diff
@@ -1,2 +1,5 @@
 # zoau
-A repository for the Z Open Automation Utilities (ZOAU) community to discuss ZOAU and provide samples and snippets 
+
+A repository for the [Z Open Automation Utilities (ZOAU)](https://www.ibm.com/docs/en/zoau) community to discuss ZOAU and provide samples and snippets.
+
+Some sample commands can be found in the [one-liners](one-liners/README.md) directory.
```

Comment:
This points to the readme as opposed to the directory. I'm thinking it might be simpler if you had text in the README.md that had a description of each longer function and then a reference to that particular script rather than just pointing them to the directory?

Reply:
Fixed.